doc/performance-investigations/0011-increase-pod-cpu-allocation/summary.md

```diff
@@ -0,0 +1,35 @@
+# Kubernetes Pod CPU Usage
+
+A number of benchmark tests were carried out to investigate the CPU usage of kubernetes pods, and how this effects Questionnaire Runner.
+
+# Summary
+
+## CPU Allocation
+
+In the k8s cluster, the allocation of CPU cores is as follows:
+
+- Each node has 3,920 mCPU allocatable.
```

Comment:
I think it's worth mentioning this is when the provisioned machine has 4 cores.

Reply:
done